### PR TITLE
Add `argmin` and `min` method in AST

### PR DESCRIPTION
**Pull request by marload**
## Description

I add `min` and `argmin`.

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

File: syft/frameworks/torch/tensors/interpreters/additive_shared.py [hunk 1]
```diff
@@ -1097,6 +1113,18 @@ def max(self, dim=None, keepdim=False, algorithm="pairwise"):
...
 
+    def min(self, dim=None, keepdim=False, algorithm="pairwise"):
+        """
+        Returns the minimun value of all elements in the input tensor, using argmin
+        Args:
+            dim: compute the min over a specific dimension
+            keepdim: keep the dimension of the tensor when dim is not None
+            algorithm: method to compute the minimum
+        Returns:
+            the max of the tensor self
```
Comment (gmuraru) [inline syft/frameworks/torch/tensors/interpreters/additive_shared.py:1124]: Here is the minimum, right?

Comment (marload) [reply syft/frameworks/torch/tensors/interpreters/additive_shared.py:1124]: Oh, it's my mistake. Thank You

File: test/torch/tensors/test_additive_shared.py [hunk 2]
```diff
@@ -860,6 +888,47 @@ def test_argmax(workers, protocol):
...
+        workers["bob"],
+        workers["james"],
+    )
+
+    args = (alice, bob)
+    kwargs = {"protocol": protocol, "crypto_provider": crypto_provider}
+
+    t = torch.tensor([3, 1.0, 2])
+    x = t.fix_prec().share(*args, **kwargs)
+    print(x.argmin)
```
Comment (gmuraru) [inline test/torch/tensors/test_additive_shared.py:905]: Remove this print

File: test/torch/tensors/test_additive_shared.py [hunk 3]
```diff
@@ -860,6 +888,47 @@ def test_argmax(workers, protocol):
...
+    t = torch.tensor([[1, 2.0, 4], [3, 9.0, 2.0]])
+    x = t.fix_prec().share(*args, **kwargs)
+    ids = x.argmin().get().float_prec()
+    assert ids.long() == torch.argmin(t)
+
+    # dim=1
+    t = torch.tensor([[1, 2.0, 4], [3, 1.0, 2.0]])
+    x = t.fix_prec().share(*args, **kwargs)
+    ids = x.argmin(dim=1).get().float_prec()
+    assert (ids.long() == torch.argmin(t, dim=1)).all()
```
Comment (gmuraru) [inline test/torch/tensors/test_additive_shared.py:929]: Q: Could you also add a test with one hot and keepdim?

Comment (marload) [reply test/torch/tensors/test_additive_shared.py:929]: sure!

File: test/torch/tensors/test_additive_shared.py (hunk 2 above)
Comment (marload) [reply test/torch/tensors/test_additive_shared.py]: complete!

File: test/torch/tensors/test_additive_shared.py (hunk 3 above)
Comment (marload) [reply test/torch/tensors/test_additive_shared.py:940]: I added it 😄

**Comment by gmuraru** [comment]
LGTM!

**Comment by marload** [comment]
Thank you for your review! 😄
